iron-gpg.c

```diff
@@ -801,6 +803,69 @@ get_user_ssh_dir(const char * const login)
 	}
 }
 
+
+/**
+ *  Copy one of the SSH key files to a .iron backup copy.
+ *
+ *  Caller can specify an empty extension, "", to copy the secret key file, or the public extension,
+ *  SSH_KEY_PUB_EXT, to copy the public key file.
+ *
+ *  It is an error if the destination file already exists.
+ *
+ *  @param ssh_dir path to directory that holds key files
+ *  @param ext extension for key file
+ *  @return int 0 if copy successful, negative number if error
+ */
+static int
+copy_ssh_key_file(const char * ssh_dir, const char * ext)
```

Comment:
I thought we already had backups of the keys. Funny to see that in the no passphrase PR.